cmd/api/src/api/v2/model.go

```diff
@@ -79,6 +79,7 @@ type DeleteSAMLProviderResponse struct {
 }
 
 type SetUserSecretRequest struct {
+	CurrentSecret      string `json:"current_secret"`
```

Comment:
We don't want the validate tag as it's possible to set initial passwords via config, also we validate later in the handler against their pw anyways